tasks/release.js

```diff
@@ -0,0 +1,64 @@
+try {
+    require( 'shelljs/global' );
+    const colors = require( 'colors' );
+    const confirm = require( 'confirm-simple' );
+    const archiver = require( 'archiver' );
+    const fs = require( 'fs' );
+} catch( e ) {
```

Comment:
True enough. I had started with something more complicated and the try/catch stayed... I'll simplify this.